cumulusci.yml

```diff
@@ -49,6 +54,23 @@ tasks:
             sql_path: datasets/data.sql
 
 flows:
+    dependencies:
+        steps:
+            1:
+                task: update_dependencies
+                options:
+                    include_beta: False
```

Comment:
Was this to get around the NPSP "mismatched versions" issue we were running into or would we prefer not to use the beta?

Reply:
Ooops! Great catch. That was from a test.

Reply:
@seanpat09 Fixed.